src/index.js

```diff
@@ -42,6 +42,8 @@ module.exports = {
       "no-multi-assign": 0,
       "no-unused-expressions": 0,
       "no-restricted-syntax": "warn",
+      "no-invalid-this": 0,
```

Comment:
was just reading about this rule... seems like one of the more complicated `eslint` rules out there.
https://eslint.org/docs/rules/no-invalid-this